source/gui/settingsDialogs.py

```diff
@@ -2123,6 +2123,10 @@ def onSave(self):
 class ReviewCursorPanel(SettingsPanel):
 	# Translators: This is the label for the review cursor settings panel.
 	title = _("Review Cursor")
+	panelDescription = _(
+		# Translators: This is descriptive text shown at the top of the Review Cursor settings panel.
+		"Configure NVDA's review cursor behavior.",
```

Comment:
@SaschaCowley I agree. However, we never really definitively say, in the user guide, what the definition of the review cursor is. The closest we come, is in the "Reviewing Text" section, where it is defined by implication.
That section talks about reviewing several kinds of thing, and then starts talking about the review cursor as "it", as if it had said "We do this by using what NVDA calls its review cursor", or some such.
We don't even link to that section from the review cursor settings sections, so maybe that is not intended as the prime definition, but it's the best I found.
cc @Qchristensen 

So maybe something like this?

```suggestion
		"Configure the review cursor, which allows reading of window contents that may otherwise be difficult to reach.",
```

Reply:
Or:
```suggestion
		"Configure the review cursor, which allows reading the contents of the screen, current document or current object without moving the caret",
```